src/code/z_demo.c

```diff
@@ -459,29 +457,27 @@ void func_800EADB0(PlayState* play, CutsceneContext* csCtx, CsCmdBase* cmd) {
                 break;
 
             case 4:
-                // func_801A246C(SEQ_PLAYER_NATURE, TYPE_1);
-                func_801A246C(SEQ_PLAYER_NATURE, 1);
+                // func_801A246C(SEQ_PLAYER_AMBIENCE, TYPE_1);
+                func_801A246C(SEQ_PLAYER_AMBIENCE, 1);
```

Comment:
Is this enum waiting for something particular?

Reply:
It'll be introduced when `func_801A246C` is decompiled. `TYPE_1` as a global macro would be too generic